tasks/main.yml

```diff
@@ -37,6 +37,25 @@
     - item.value.state == 'present'
     - not item.key ~ '@' ~ item.value.domain in register_sasldblistusers.stdout_lines
 
+- name: Set cyrus_sasl_sasldb_file_real_path
+  set_fact:
+    # BSDs appends '.db' to the given file name, but others treat it as-is
+    cyrus_sasl_sasldb_file_real_path: "{% if ansible_os_family == 'FreeBSD' or ansible_os_family == 'OpenBSD' %}{{ cyrus_sasl_sasldb_file }}.db{% else %}{{ cyrus_sasl_sasldb_file }}{% endif %}"
```

Comment:
need an explanation why adding .db to __cyrus_sasl_sasldb_file in vars/*BSD.yml does not work.
'cyrus_sasl_saslpassword_command' and '/usr/local/bin/sasl_check_pw' works without .db extension?